spec/policies/institution_policy_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
```

Comment:
You can create a `.rspec` (in the root of the project) containing `--require rails_helper`, then you won't need to require it in every file :)